### ISSUE TITLE
SE server no longer appears in region selection

### ISSUE THREAD

**Issue by SethDusek** [ValveSoftware/Dota2-Gameplay#7855]
### Description

Hi, since the morning I've been unable to queue for matches in SE Asia. The region selection doesn't show it:
![image](https://user-images.githubusercontent.com/9436380/230037095-35cee112-93c7-42be-bc3c-4f5f4fdb99eb.png)

I initially thought maybe my ping to SEA was too high and that's why it's not appearing, but I can select Singapore server in lobbies and I get 98 ping (compared to 140-ish EUW and EUE, the only other regions I can play ranked).

**Comment by commentator24** [comment]
i also have US east missing. just realised after you post this 
![image](https://user-images.githubusercontent.com/109324155/230081392-6077248b-ca50-4fda-a142-d5513f75ca85.png)

**Comment by Aluerie** [comment]
this^ has been for years that people don't have one random red-bad server in their list. For example, I'm missing Chile server since forever in my list. It's probably intended versus some lobby abuse, maybe not ? hard to judge.  

However, the fact that this missing region is supposedly very good for OP is very weird.

**Comment by SethDusek** [comment]
> However, the fact that the missing region is supposedly very good for OP is very weird.

For some reason the regional chat is also placing me in Quito, in Ecuador. If I go to the list of regional channels, it shows the ones in my country being thousands of kilometres away. I'm not using any VPN.

It seems that I have found the issue. For some reason Dota 2 is placing me in a region thousands of kilometres away from where I actually live, thus removing SE Asia from the list.

If I open my VPN and set my region to Japan, SE Asia now pops up on the list. As a workaround while this issue is being fixed, I guess I will turn on my VPN, queue a match in SEA, and then once it's found stop the VPN.

This is especially strange since IP geolocation websites place me in the correct country, but for some reason Dota 2 isn't.

**Comment by QMHRF** [comment]
same is happening with me... any solutions?

**Comment by SethDusek** [comment]
> same is happening with me... any solutions?

Please check my above comment and see if it can help you too. First try to see where you're being placed in regional chat. If it's not placing you in the correct place then try using a VPN that can place you somewhere in SEA (Singapore, Japan, etc.

**Comment by QMHRF** [comment]
alright bro

**Comment by QMHRF** [comment]
it worked bro

**Comment by Aluerie** [comment]
> the regional chat is also placing me in Quito, in Ecuador

yeah, another weird thing in your list is that it should be sorted like in commentator24's screenshot: green first, red last (and very last red should be missing). Just another evidence that Dota thinks you are somewhere in SA.

**Comment by moizahmedq** [comment]
bump

**Comment by SethDusek** [comment]
> bump

Please react with a thumbs-up on the original comment. Also try my solution to see if it works for you

**Comment by moizahmedq** [comment]
> > bump
> 
> Please react with a thumbs-up on the original comment. Also try my solution to see if it works for you

what vpn did u use
i dont have any

**Comment by SethDusek** [comment]
> > > bump
> > 
> > 
> > Please react with a thumbs-up on the original comment. Also try my solution to see if it works for you
> 
> what vpn did u use i dont have any

Try to find a good free VPN. TunnelBear might work.

**Comment by moizahmedq** [comment]
> > > > bump
> > > 
> > > 
> > > Please react with a thumbs-up on the original comment. Also try my solution to see if it works for you
> > 
> > 
> > what vpn did u use i dont have any
> 
> Try to find a good free VPN. TunnelBear might work.

yup it worked

**Comment by SethDusek** [comment]
It's been a month and no resolution yet. In the meantime I've found a solution which is to force a relay cluster (as far as I can tell, this changes your routing to the server, I use this for reducing ping sometimes). 

If I type net_option SDRClient_ForceRelayCluster sgp into the console and queue automatically selected region, it queues SEA. Depending on which server you want to queue you will need something other than sgp (Singapore). 

Here is the list of relay clusters I found:
>dxb=47+4,bom=187+18/73+4,maa=127+12/90+4,sgp=97+9/98+9,hkg=130+13/127+9,par=140+14,fra=141+14,lhr=149+14/147+14,ams=149+14/148+14,vie=154+15/152+14,iad=205+20,gru=328+32/320+20

For a seperate bug I should create, the SDRClient_ForceRelayCluster no longer works ingame (it used to). Sometimes my game defaults to dxb which has 120 ping, so I used to use the command to force it to use sgp (for 98-ish ping). Now I need to restart dota, type in the command and then reconnect
